javascript/src/test/pages/Home/Home.test.js

```diff
@@ -1,28 +1,101 @@
 import React from "react";
-import { render } from "@testing-library/react";
-import Home from "main/pages/Home/Home";
+import { Jumbotron } from "react-bootstrap";
+import { waitFor, render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Home from "main/pages/Home/Home"
 import useSWR from "swr";
 jest.mock("swr");
+import { fetchWithoutToken } from "main/utils/fetch";
+jest.mock("main/utils/fetch");
+import { useHistory } from "react-router-dom";
+
 
 describe("Home tests", () => {
-  const filterData = [
-    {
-      id: "1",
-      activeQuarter:"something that exists that isnt all"}
-  ];
+	const courses = [
+	  {
+	    name: "CMPSC 156",
+	    id: 1,
+	    quarter: "F20",
+	    instructorFirstName: "Phill",
+	    instructorLastName: "Conrad",
+	    instructorEmail: "phtcon@ucsb.edu",
+	  },
+	  {
+	    name: "CMPSC 148",
+	    id: 2,
+	    quarter: "W21",
+	    instructorFirstName: "Chandra",
+	    instructorLastName: "Krintz",
+	    instructorEmail: "krintz@example.org",
+	  },
+	];
+
+	const filterdata = [
+	  {
+	    id: "2",
+	    activeQuarter: "All"
+	  }
+	];
+
+	const filterData = [
+	  {
+	    id: "1",
+	    activeQuarter:"F20"}
+	];
```

Comment:
Can we rename one of these variables? These two names differ only in casing, which is very confusing.